stock_auto_move/stock_auto_move.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf8 -*-
+#
+# Copyright (C) 2014 NDP Systèmes (<http://www.ndp-systemes.fr>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from openerp import fields, models, api
+
+class stock_auto_move_move(models.Model):
```

Comment:
This module is not PEP8 compliant. I let you fix the code format otherwise it wont be accepted.